### PR TITLE
Playlists

### PR DESCRIPTION
**Pull request by mitchdowney**

### CODE REVIEW

File: Podverse/Playlist.swift [hunk 1]
```diff
@@ -9,53 +9,145 @@
...
-//                }
-//            }
-//            if let clips = clips {
-//                for clip in clips {
-//                    allItemsArray.append(clip)
-//                }
-//            }
+    static func retrievePlaylistFromServer(id: String, completion: @escaping (_ playlist: Playlist?) -> Void) {
+        
+        if let url = URL(string: "http://localhost:8080/api/playlist") {
```
Comment (kreonjr) [inline Podverse/Playlist.swift]: Suggestion for this URL (and possibly in other locations as well):
Have 3 constants in the Constants file:
```
let DEV_URL = "http://localhost:8080/api/"
let PROD_URL = "whatever the non localhost url will be"
let BASE_URL = DEV_URL
```
Then replace all places in the app with a url with the BASE_URL. This way, when we need to switch urls, we only need to switch it out it in the constants file and set the BASE_URL to whatever we want. And use it as such: `URL(string: BASE_URL + "playlist")`

File: Podverse/Playlist.swift [hunk 2]
```diff
@@ -9,53 +9,145 @@
...
+                            
+                        }
+                        
+                        DispatchQueue.main.async {
+                            completion(playlists)
+                        }
+                        
+                    } catch {
+                        print(error)
+                        print("Error")
```
Comment (kreonjr) [inline Podverse/Playlist.swift]: Probably should make this one print and a little more descriptive: ```print("Error: \(error.localizedDescription)")```

File: Podverse/PlaylistDetailTableViewController.swift [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let mediaRef = self.mediaRefsArray[indexPath.row]
+        let cell = self.tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! PlaylistDetailTableViewCell
+
+        cell.episodeTitle.text = mediaRef.episodeTitle
+        cell.podcastTitle.text = mediaRef.podcastTitle
+        cell.pubDate.text = mediaRef.episodePubDate?.toShortFormatString()
+        
+        Podcast.retrievePodcastImage(podcastImageURLString: mediaRef.podcastImageUrl) { (podcastImage) -> Void in
+            DispatchQueue.main.async {
```
Comment (kreonjr) [inline Podverse/PlaylistDetailTableViewController.swift]: This is probably not needed any more since retrieve dispatches to main queue now anyway

**Comment by mitchdowney** [comment]
Ready for re-review

**Pull request merged**